src/openforms/authentication/contrib/yivi_oidc/oidc_plugins/plugins.py

```diff
@@ -388,8 +387,10 @@ def _get_signicat_yivi_condiscon_scope(self, options: YiviOptions) -> str:
         base64_bytes = base64.b64encode(condiscon_string.encode("ascii"))
         return f"signicat:param:condiscon_base64:{base64_bytes.decode('ascii')}"
 
-    # TODO: parent has django.http.HttpRequest, but it should be rest_framework.request.Request (maykinmedia/mozilla-django-oidc-db/issues/151)
-    def get_extra_params(self, request: Request, extra_params: dict) -> dict:  # pyright: ignore[reportIncompatibleMethodOverride]
+    def get_extra_params(
+        self, request: HttpRequest, extra_params: dict
+    ) -> dict[str, str | bytes]:
+        assert isinstance(request, Request)  # stronger guarantee
```

Comment:
Ahh I see, i was struggling because HttpRequest doesn't have the query_params. But so this is the way to do it. :nerd_face: